packages/origin-247-certificate/migrations/1638376770098-CreateOffchainCertificateTables.ts

```diff
@@ -1,7 +1,7 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
 
-export class createOffchainCertificateTables1638376770098 implements MigrationInterface {
-    name = 'createOffchainCertificateTables1638376770098';
+export class CreateOffchainCertificateTables1638376770098 implements MigrationInterface {
```

Comment:
It somehow didn't detect the migration with lowercase, but I guess it's okay as we use capitalized migration names everywhere else